### PR TITLE
LPS-80935 Persist the events with their respective user id

### PR DESCRIPTION
**Pull request by igormatos**

### CODE REVIEW

File: analytics-core/src/main/java/com/liferay/analytics/android/util/FlushProcess.kt [hunk 1]
```diff
@@ -41,27 +41,24 @@ internal class FlushProcess(fileStorage: FileStorage, private var flushInterval:
...
-
-	fun getRemainingEvents(): List<Event> {
-		return eventsDAO.getEvents().drop(FLUSH_SIZE)
-	}
-
 	private fun saveEventsQueue() {
-		eventsDAO.addEvents(eventsQueue)
+		eventsQueue.forEach { (userId, events) ->
+			eventsDAO.addEvents(userId, events)
+		}
```
Comment (victorg1991) [inline analytics-core/src/main/java/com/liferay/analytics/android/util/FlushProcess.kt]: eventsQueue.forEach(eventsDAO::addEvents) :P

File: analytics-core/src/main/java/com/liferay/analytics/android/dao/EventsDAO.kt [hunk 2]
```diff
@@ -27,31 +27,32 @@ import java.lang.reflect.Type
  */
 internal class EventsDAO(private var fileStorage: FileStorage) {
 
-	fun getEvents(): List<Event> {
+	fun getEvents(): HashMap<String, List<Event>> {
```
Comment (victorg1991) [inline analytics-core/src/main/java/com/liferay/analytics/android/dao/EventsDAO.kt]: Why not using Map?

**Comment by allanragec** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by allanragec** [comment]
Thank you, pull request merged! [See changes here.](https://github.com/liferay-mobile/liferay-analytics-android/compare/6e5c7d0~3...6e5c7d0) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**